README.md

```diff
@@ -106,6 +106,10 @@ all:
 ### 3. Setup playbook variables
 
 Create a vars.json file, following the example below:
+> **Note**: <br />
+> For a fully functional deployment be sure to use either an **external IP** or an **internal IP** for `rabbitmq_host` based on the infra network configuration.
+> 
+> Additionally, retrieving the actual api-key from the server is not supported yet, so use `"enable_api_key": "false"` in extra vars as any value in `trento_api_key` would be ineffective.
```

Comment:
I don't understand very well why we are moving these note to this location. As a matter of fact, I don't understand the need for the note at all. I never had to set parameter rabbitmq_host in any of my tests. 

I don't understand either the remark about the enable_api_key. It almost sounds to me as if we are suggesting to disable the API the key.

Reply:
Hey @abravosuse, I'll further improve these based on your comments but some context first:

>I don't understand very well why we are moving this note to this location.
This is simply because it's closer to the section where we tell the user to create the configuration file where this would be defined.
>  I don't understand the need for the note at all. I never had to set parameter rabbitmq_host in any of my tests.
by default it uses either `localhost` when using `rpm` install method and `host.docker.internal` when using the `docker` install method, which works for single host deployments. The note here I think was originally intended top let the user know that he might require to set the value manually depending on his network configuration and layout of the deployment. I'll improve that and write something in along those lines

>I don't understand either the remark about the enable_api_key parameter. It almost sounds to me as if we are suggesting to disable the API the key.
about this, what it means is that if the user wants to also deploy agents from the playbook, he will have to do so in two stages (first install the server, manually get the API key, then add it to the vars and redeploy also enabling agent deployment OR disable the api_key completely to allow for deployment in a single run. I'll also expand a bit more on that.